webapp-django/crashstats/documentation/jinja2/documentation/docs_base.html

```diff
@@ -0,0 +1,56 @@
+{% extends "crashstats_base.html" %}
+
+{% macro navlink(link_url, link_name) %}
+<li>
+    <a href="{{ link_url }}" {% if request.path == link_url %}class="selected"{% endif %}>{{ link_name }}</a>
+</li>
+{% endmacro %}
+
+
+{% block site_css %}
+{{ super() }}
+{% stylesheet 'documentation' %}
+{% stylesheet 'jsonview' %}
+{% endblock %}
+
+{% block site_js %}
+{{ super() }}
+{% javascript 'jsonview' %}
+{% javascript 'documentation' %}
+{% endblock %}
+
+{% block page_title%}Crash Stats Documentation: {% block doc_title %}Home{% endblock %}{% endblock %}
+
+{% block content %}
+<div id="mainbody">
+    <div class="page-heading">
+        <h2>{{ self.page_title() }}</h2>
```

Comment:
`page_title` here is an attribute on the template that pulls in the `page_title` block value. That's how you use block values multiple times in the same Jinja2 template.